### DISCUSSION TITLE
Går inte att ladda ned från SVT-play idag

### DISCUSSION THREAD

**Discussion by 68456gjp49586g** [stefansundin/privatkopiera#141] [Q&A] [answered]
Inget går att ladda ned idag (24/11), får svaret: "Fel: Den här hemsidan stöds ej."
Senast 20/11 fungerade det.

**Comment by stefansundin** [comment, marked as answer]
Det här ska nu vara fixat i v0.3.8. Förlåt att det dröjde. Läs mer här: https://github.com/stefansundin/privatkopiera/discussions/142

**Reply by Shroom86** [reply]
Tusen tack! 🙂
